Models/EmployeeStatus.cs

```diff
@@ -3,8 +3,8 @@
     public enum EmployeeStatusId
     {
         Employed = 1,
-        Vacationing,
-        Fired
+        Vacationing = 2,
+        Fired = 3
```

Comment:
There is no need to specify values `2` and `3` explicitly here as they are already `2` and `3`  by default.

Reply:
Changed